br/backup-and-restore-tool.md

```diff
@@ -20,10 +20,21 @@ aliases: ['/docs/dev/br/backup-and-restore-tool/','/docs/dev/reference/tools/br/
 - It is recommended that you deploy BR on the PD node.
 - It is recommended that you mount a high-performance SSD to BR nodes and all TiKV nodes. A 10-gigabit network card is recommended. Otherwise, bandwidth is likely to be the performance bottleneck during the backup and restore process.
 
-## Download Binary
+> **Note:**
+>
+> If you don't mount a network disk or use other shared storage, the BR backup data will be generated on each TiKV node. Because BR only backs up the leader replica, the space reserved for each node needs to be estimated based on the leader size.
+> Meanwhile, because TiDB v4.0 uses leader count for balancing by default, there will be a problem of large differences in leader size, resulting in uneven backup data for each node.
```

Comment:
```suggestion
> Meanwhile, because TiDB v4.0 uses leader count for load balancing by default, leaders are greatly different in size, resulting in uneven distribution of backup data on each node.
```